### PR TITLE
Themed Spotlights news post

### PR DESCRIPTION
**Pull request by Shadowthegod**
### Description

Article for the Anime Remix Themed Spotlights. I'm actually quite unsure how it looks like because it would look much better with images instead of the name of the modes. Might just be me though.

### Status

- [ ] WIP
- [ ] Pending Review
- [ ] Finished

### CODE REVIEW

File: news/2018-03-05-anime-remix-themed-spotlights-2018.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+date: 2018-03-05 14:00:00 +0000
+---
+
+Greetings everyone! We are happy to announce that the Themed Spotlights are back! To celebrate this we prepared a special selection of anime remixes for you! Enjoy the songs while clicking those circles, hitting the drum, catching those healthy fruits and go octopus-mode with the squares!
+
+![](/wiki/shared/news/2018-03-05-anime-remix-themed-spotlights-2018/spotlight-header.jpg)
+
+With that said, we just released the Anime Remix Themed Charts and can be found [here](https://osu.ppy.sh/p/chart?ch=ARCH17). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access.
+
+##osu!
```
Comment (MegaApplePi) [inline news/2018-03-05-anime-remix-themed-spotlights-2018.md]: add a space between `##` and `osu!`
(repeat for other headings too)

Comment (Shadowthegod) [reply news/2018-03-05-anime-remix-themed-spotlights-2018.md]: Worst mistake ever. (╯°□°）╯︵ ┻━┻

File: news/2018-03-05-anime-remix-themed-spotlights-2018.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Anime Remix Themed Spotlights 2018"
+date: 2018-03-05 14:00:00 +0000
+---
+
+Greetings everyone! We are happy to announce that the Themed Spotlights are back! To celebrate this we prepared a special selection of anime remixes for you! Enjoy the songs while clicking those circles, hitting the drum, catching those healthy fruits and go octopus-mode with the squares!
```
Comment (TPGPL) [inline news/2018-03-05-anime-remix-themed-spotlights-2018.md]: comma after this
the drum -> the drums
going octopus mode with the notes

File: news/2018-03-05-anime-remix-themed-spotlights-2018.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+![](/wiki/shared/news/2018-03-05-anime-remix-themed-spotlights-2018/spotlight-header.jpg)
+
+With that said, we just released the Anime Remix Themed Charts and can be found [here](https://osu.ppy.sh/p/chart?ch=ARCH17). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access.
```
Comment (TPGPL) [inline news/2018-03-05-anime-remix-themed-spotlights-2018.md]: we have just released the Anime Remix Themed Charts which can be found

File: news/2018-03-05-anime-remix-themed-spotlights-2018.md (hunk 2 above)
Comment (Shadowthegod) [reply news/2018-03-05-anime-remix-themed-spotlights-2018.md]: Fixed them, although technically in taiko you only use one drum. 🤔

**Pull request closed without merging**